### PR TITLE
mkdir recursively to fix #6

### LINKED ISSUE #6: crawl site missing 'output' folder
I received the error below. I could verify the script was trying to mkdir, but it wouldn't work. After manually creating the folder 'output'. The script worked as expected. Unsure if this is a bug or user error...

`UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir 'output/https___news.polymer-project.org_'`

### PR DESCRIPTION
**Pull request by EnixCoda**
Hi,
I played few examples of this repo and had same issue as #6 have described.
Tested on my local.
Hope it helps.

Fixes #6

### CODE REVIEW

**Comment by EnixCoda** [comment]
I signed it!

**Comment by misterhtmlcss** [comment]
@EnixCoda i forked the software. feel free to add your tweak your code to run with mine. I'll definitely allow your PRs in a timely manner ;)

File: crawlsite.js [hunk 1]
```diff
@@ -56,7 +56,11 @@ function slugify(str) {
 
 function mkdirSync(dirPath) {
   try {
-    fs.mkdirSync(dirPath);
+    dirPath.split('/').reduce((parentPath, dirName) => {
+      const currentPath = parentPath + dirName;
+      if (!fs.existsSync(currentPath)) fs.mkdirSync(currentPath);
```
Comment (ebidel) [inline crawlsite.js]: use `{}` and 2 space indent on the return.

File: crawlsite.js [hunk 2]
```diff
@@ -56,7 +56,11 @@ function slugify(str) {
...
+      return currentPath + '/';
+    }, '')
```
Comment (ebidel) [inline crawlsite.js]: `;`

**Comment by EnixCoda** [comment]
I'm unable to edit on this PR anymore due to my fork was deleted.

I'll create a new PR.

**Comment by EnixCoda** [comment]
#16

**Pull request closed without merging**